### PR TITLE
SCUMM: MANIAC NES: Add WIP loading for NES title screens. Not drawing them yet.

### PR DESCRIPTION
**Pull request by robertmegone**
Adds resource parsing for the NES title screens, currently I'm sticking these at the end of LFL 54, though I'm not quite sure exactly where or how to load them.

### CODE REVIEW

File: engines/scumm/file_nes.cpp [hunk 1]
```diff
@@ -126,6 +137,7 @@ const ScummNESFile::ResourceGroup res_roomgfx = {
 	res_roomgfx_ger,
 	res_roomgfx_esp,
 	res_roomgfx_ita,
+    res_roomgfx_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:140]:
```suggestion
	res_roomgfx_pro,
```

File: engines/scumm/file_nes.cpp [hunk 2]
```diff
@@ -147,6 +160,7 @@ const ScummNESFile::ResourceGroup res_costumegfx = {
 	res_costumegfx_ger,
 	res_costumegfx_esp,
 	res_costumegfx_ita,
+    res_costumegfx_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:163]:
```suggestion
	res_costumegfx_pro,
```

File: engines/scumm/file_nes.cpp [hunk 3]
```diff
@@ -252,6 +278,7 @@ const ScummNESFile::ResourceGroup res_rooms = {
 	res_rooms_ger,
 	res_rooms_esp,
 	res_rooms_ita,
+    res_rooms_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:281]:
```suggestion
	res_rooms_pro,
```

File: engines/scumm/file_nes.cpp [hunk 4]
```diff
@@ -532,6 +598,7 @@ const ScummNESFile::ResourceGroup res_scripts = {
 	res_scripts_ger,
 	res_scripts_esp,
 	res_scripts_ita,
+    res_scripts_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:601]:
```suggestion
	res_scripts_pro,
```

File: engines/scumm/file_nes.cpp [hunk 5]
```diff
@@ -742,6 +835,7 @@ const ScummNESFile::ResourceGroup res_costumes = {
 	res_costumes_ger,
 	res_costumes_esp,
 	res_costumes_ita,
+    res_costumes_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:838]:
```suggestion
	res_costumes_pro,
```

File: engines/scumm/file_nes.cpp [hunk 6]
```diff
@@ -763,6 +858,7 @@ const ScummNESFile::ResourceGroup res_globdata = {
 	res_globdata_ger,
 	res_globdata_esp,
 	res_globdata_ita,
+    res_globdata_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:861]:
```suggestion
	res_globdata_pro,
```

File: engines/scumm/file_nes.cpp [hunk 7]
```diff
@@ -785,6 +882,7 @@ const ScummNESFile::ResourceGroup res_sprpals = {
 	res_sprpals_ger,
 	res_sprpals_esp,
 	res_sprpals_ita,
+    res_sprpals_pro
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:885]:
```suggestion
	res_sprpals_pro
```

File: engines/scumm/file_nes.cpp [hunk 8]
```diff
@@ -807,6 +906,7 @@ const ScummNESFile::ResourceGroup res_sprdesc = {
 	res_sprdesc_ger,
 	res_sprdesc_esp,
 	res_sprdesc_ita,
+    res_sprdesc_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:909]:
```suggestion
	res_sprdesc_pro,
```

File: engines/scumm/file_nes.cpp [hunk 9]
```diff
@@ -851,6 +954,7 @@ const ScummNESFile::ResourceGroup res_sproffs = {
 	res_sproffs_ger,
 	res_sproffs_esp,
 	res_sproffs_ita,
+    res_sproffs_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:957]:
```suggestion
	res_sproffs_pro,
```

File: engines/scumm/file_nes.cpp [hunk 10]
```diff
@@ -873,6 +978,7 @@ const ScummNESFile::ResourceGroup res_sprdata = {
 	res_sprdata_ger,
 	res_sprdata_esp,
 	res_sprdata_ita,
+    res_sprdata_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:981]:
```suggestion
	res_sprdata_pro,
```

File: engines/scumm/file_nes.cpp [hunk 11]
```diff
@@ -894,6 +1001,7 @@ const ScummNESFile::ResourceGroup res_charset = {
 	res_charset_ger,
 	res_charset_esp,
 	res_charset_ita,
+    res_charset_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:1004]:
```suggestion
	res_charset_pro,
```

File: engines/scumm/file_nes.cpp [hunk 12]
```diff
@@ -915,9 +1024,33 @@ const ScummNESFile::ResourceGroup res_preplist = {
 	res_preplist_ger,
 	res_preplist_esp,
 	res_preplist_ita,
+    res_preplist_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:1027]:
```suggestion
	res_preplist_pro,
```

File: engines/scumm/file_nes.h [hunk 13]
```diff
@@ -36,6 +36,7 @@ class ScummNESFile : public BaseScummFile {
 		kROMsetGermany,
 		kROMsetSpain,
 		kROMsetItaly,
+        kROMsetPro,
```
Comment (bluegr) [inline engines/scumm/file_nes.h:39]:
```suggestion
		kROMsetPro,
```

File: engines/scumm/file_nes.h [hunk 14]
```diff
@@ -59,7 +60,8 @@ class ScummNESFile : public BaseScummFile {
 		NES_SPROFFS,
 		NES_SPRDATA,
 		NES_CHARSET,
-		NES_PREPLIST
+		NES_PREPLIST,
+        NES_TITLES
```
Comment (bluegr) [inline engines/scumm/file_nes.h:64]:
```suggestion
		NES_TITLES
```

**Comment by bluegr** [comment]
@robertmegone thanks!

There are some whitespace issues, which I've fixed
Also, the first commit does not conform to our guidelines

File: engines/scumm/file_nes.cpp [hunk 15]
```diff
@@ -679,6 +764,7 @@ const ScummNESFile::ResourceGroup res_sounds = {
 	res_sounds_ger,
 	res_sounds_esp,
 	res_sounds_ita,
+    res_sounds_pro,
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:767]:
```suggestion
	res_sounds_pro,
```

File: engines/scumm/file_nes.cpp [hunk 16]
```diff
@@ -829,6 +930,7 @@ const ScummNESFile::ResourceGroup res_sprlens = {
 	res_sprlens_ger,
 	res_sprlens_esp,
 	res_sprlens_ita,
+    res_sprlens_pro
```
Comment (bluegr) [inline engines/scumm/file_nes.cpp:933]:
```suggestion
	res_sprlens_pro
```

**Comment by robertmegone** [comment]
> @robertmegone thanks!
> 
> There are some whitespace issues, which I've fixed Also, the first commit does not conform to our guidelines

Thanks! I've reworded the first commit.

**Comment by bluegr** [comment]
Excellent work, thanks!

**Pull request closed without merging**